metrics/prometheus.go

```diff
@@ -1212,6 +1213,36 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 			}
 		}
 	}
+
+	if c.includedMetrics.Has(container.AppMetrics) {
```

Comment:
can we move this up with other metrics?  That way we get consistent labels with other metrics.

Reply:
I think not, because these metrics are initialized when cadvisor starts, but application metrics can be changed at any time

Reply:
I see.  You don't need to use the containerMetrics thingy above, but if you place it after `for _, cm := range c.containerMetrics {`, you can simplify your function a bit.